projects/exui-common-lib/src/lib/gov-ui/components/gov-uk-input/gov-uk-input.component.html

```diff
@@ -4,12 +4,17 @@
 <xuilib-gov-label [config]="config"></xuilib-gov-label>
 
 <span *ngIf="config.hint" [id]="config.id +'-hint'" class="govuk-hint">
- {{config.hint}}
+  {{config.hint}}
 </span>
 
-<xuilib-gov-uk-error-message [config]="config" [errorMessage]="errorMessage"></xuilib-gov-uk-error-message>
+<xuilib-gov-uk-error-message [config]="config" [errorMessage]="errorMessage" *ngIf="errorMessage"></xuilib-gov-uk-error-message>
```

Comment:
@johnbenjamin-hmcts - probably no harm leaving the `*ngIf` here, but is there ever a scenario where `errorMessage` is provided but is falsy? In other words, would there ever be an empty error message?